### DISCUSSION TITLE
Not Receiving Data from Marketing Stream

### DISCUSSION THREAD

**Discussion by vipul-neon** [amzn/ads-advanced-tools-docs#16] [Q&A] [answered]
Hi,

I have successfully subscribed to sp-traffic and sp-conversion datasetIds. The status shown is Active. I am still not receiving any data.
Is there anything else (apart from the sqs queue) that is needed to be setup? 
Please help in this regard

**Comment by kdwivedi13** [comment, marked as answer]
If you have successfully subscribed to data sets sp-traffic and sp-conversion and subscription is in active state, then there is no other step required. However, It is worth knowing that you will only receive the data if there are impressions. And I hope you are not subscribed through a DSP Profile. Because in current state, DSP Profile is not supported.
